src/wrapper/wrapper.c

```diff
@@ -169,16 +112,16 @@ void dyad_wrapper_fini ()
     dyad_finalize (&ctx);
 }
 
-int open (const char *path, int oflag, ...)
+DYAD_DLL_EXPORTED int open (const char *path, int oflag, ...)
 {
     char *error = NULL;
     typedef int (*open_ptr_t) (const char *, int, mode_t, ...);
     open_ptr_t func_ptr = NULL;
     int mode = 0;
 
-    if (ctx == NULL) {
-        dyad_wrapper_init ();
-    }
+    // if (ctx == NULL) {
+    //     dyad_wrapper_init ();
+    // }
```

Comment:
I guess this logic is commented as the init happens on the library load. If so remove the commented lines.

Reply:
Thanks for catching this. I meant to remove it, but it must have slipped under my radar. I'll fix that.

Reply:
Done